app/controllers/conference_registrations_controller.rb

```diff
@@ -20,6 +20,11 @@ def new
       redirect_to edit_conference_conference_registrations_path(@conference.short_title)
     end
 
+    if @conference.registration_limit_exceeded?
```

Comment:
Anyway, we can discuss if the validation belongs to conference or registration! At least I would like to have a validation which prohibits to have more registrations than specified in :registration_limit attribute, either in registration or conference model! 

The other PR is merged!